src/main/resources/templates/db_scripts/schema.json

```diff
@@ -34,7 +34,13 @@
       "generateId": true,
       "fromModuleVersion": 1.0,
       "withMetadata": true,
-      "populateJsonWithId": true
+      "populateJsonWithId": true,
+      "foreignKeys": [
```

Comment:
I know that this ticket is intended to cover only voucher and voucher lines relations but it would be nice to have the same for invoice line ust to not come back to it later...